<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string>background-color: rgb(118, 118, 245);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nameInp">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(118, 118, 245);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string>background-color: rgb(118, 118, 245);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
      </property>
      <property name="text">
       <string>Count</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="ageInp">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(118, 118, 245);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string>background-color: rgb(118, 118, 245);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
      </property>
      <property name="text">
       <string>Price</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="bragsPerDayInp">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(118, 118, 245);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string>background-color: rgb(118, 118, 245);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
      </property>
      <property name="text">
       <string>Size</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="creditRatingInp">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(118, 118, 245);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string>background-color: rgb(118, 118, 245);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
      </property>
      <property name="text">
       <string>is avaliable</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="evilInp">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(118, 118, 245);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string>background-color: rgb(118, 118, 245);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
      </property>
      <property name="text">
       <string>lenght_1</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="networth0Inp">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(118, 118, 245);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(118, 118, 245);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
      </property>
      <property name="text">
       <string>lenght_2</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDoubleSpinBox" name="networth1Inp">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(118, 118, 245);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string>background-color: rgb(118, 118, 245);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
      </property>
      <property name="text">
       <string>lenght_3</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QDoubleSpinBox" name="networth2Inp">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(118, 118, 245);
color: rgb(255, 255, 255);
font: 500 16pt &quot;Futura&quot;;
border-radius: 10px;
padding: 6px;</string>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000000.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>764</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
